### PR TITLE
Added new feature  for remove accents. Added test to remove accents

### PR DESCRIPTION
**Pull request by Bernardoow**
I just create a new feature for remove accents. For some work I need cleaner string.

Maybe it can be useful for community.

### CODE REVIEW

**Comment by Bernardoow** [comment]
I don't know why check fails.

File: tests/Example.elm [hunk 1]
```diff
@@ -0,0 +1,10 @@
+module Example exposing (..)
+
+import Test exposing (..)
+import Expect
+import Fuzz exposing (list, int, string)
+
+
+suite : Test
+suite =
+    todo "Implement our first test. See http://package.elm-lang.org/packages/elm-community/elm-test/latest for how to do this!"
```
Comment (lorenzo) [inline tests/Example.elm]: I don't think we need this file

File: tests/elm-package.json [hunk 2]
```diff
@@ -1,19 +1,16 @@
...
     "license": "BSD3",
     "source-directories": [
-        ".",
-        "../src"
+        "../src",
+        "."
     ],
     "exposed-modules": [],
     "dependencies": {
-        "elm-community/elm-test": "3.0.0 <= v < 4.0.0",
```
Comment (lorenzo) [inline tests/elm-package.json]: What happened here?

File: src/String/Extra.elm [hunk 3]
```diff
@@ -959,3 +960,40 @@ nonEmpty string =
...
+                , ( "ñ", "n" )
+                , ( "Ñ", "N" )
+                , ( "[ò-ö]", "o" )
+                , ( "[Ò-Ö]", "O" )
+                , ( "[ù-ü]", "u" )
+                , ( "[Ù-Ü]", "U" )
+                , ( "ý", "y" )
+                , ( "ÿ", "y" )
+                , ( "Ý", "Y" )
+                ]
```
Comment (lorenzo) [inline src/String/Extra.elm]: Could you move this to a top level function? The ideally we would build the `Regex` only once, and not once per each call to the function:

```elm
accentRegex : List Regex
accentRegex = 
    ...
```

File: tests/RemoveAccentsTest.elm [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+
+
+removeAccentsTest : Test
+removeAccentsTest =
+    describe "removeAccents"
+        [ test "Should result string without acents" <|
+            \() ->
+                removeAccents "áàãâäéèêëíìîïóòõôöúùûüçÁÀÃÂÄÉÈÊËÍÌÎÏÓÒÕÖÔÚÙÛÜÇ"
+                    |> Expect.equal "aaaaaeeeeiiiiooooouuuucAAAAAEEEEIIIIOOOOOUUUUC"
+        , test "Should result in string without acents" <|
```
Comment (prikhi) [inline tests/RemoveAccentsTest.elm]: Latest versions of elm-test require that each `test` is given a unique string.

Also, `acents` -> `accents`

**Comment by prikhi** [comment]
Make sure you're running the tests locally w/ elm-test 0.18.5

File: tests/RemoveAccentsTest.elm (hunk 4 above)
Comment (Bernardoow) [inline tests/RemoveAccentsTest.elm]: OK. I will fix it.

File: src/String/Extra.elm (hunk 3 above)
Comment (Bernardoow) [inline src/String/Extra.elm]: OK ! I will do it.

**Comment by Bernardoow** [comment]
We need to update test of this package because with lasted version of elm-test the check fails. With elm-test@0.18.0 it's run and pass in test.

**Comment by lorenzo** [comment]
Great job! Thanks for your contribution :)

**Pull request merged**